openapi/components/schemas/Common/CommonAlternativeInstrument.yaml

```
@@ -10,6 +11,13 @@ properties:
     description: The method of payment instrument.
     allOf:
       - $ref: ../PaymentMethods/AlternativePaymentMethods.yaml
+      - not: 
+          enum:
+            - payment-card
+            - paypal
+            - ach
+            - echeck
+            - Khelocard
   billingAddress:
     description: The billing address.
```

nitpick: suggestion to reduce ambiguity
```suggestion
    description: Billing address of the user that is associated with the payment instrument.
```